packages/exceptions/index.ts

```diff
@@ -0,0 +1,20 @@
+export * from './src/assert/AssertionException';
+export * from './src/base/Exception';
+export * from './src/base/RuntimeException';
+export * from './src/collections/NoSuchItemException';
+export * from './src/collections/IndexOutOfBoundsException';
+export * from './src/collections/EmptyQueueException';
+export * from './src/collections/EmptyStackException';
+export * from './src/format/FormatException';
+export * from './src/format/VersionFormatException';
+export * from './src/module/ClassLoaderException';
+export * from './src/text/ParsingException';
+export * from './src/uri/UriFormatException';
+export * from './src/uri/UriIntegrityException';
+export * from './src/InvalidArgumentException';
+export * from './src/InvalidOperationException';
+export * from './src/InvalidStateException';
+export * from './src/MethodNotImplementedException';
+export * from './src/MissingKeyException';
+export * from './src/OperationNotSupportedException';
+export * from './src/RangeException';
```

Comment:
I think `RangeException` should be removed / deprecated because it's usage scope is not clear.